config/initializers/markdown.rb

```diff
@@ -2,8 +2,13 @@
 require 'rouge'
 
 class Markdown
+  LINK_REGEXP = %r((?<![=["|']|><a-z0-9_\.])(?<!http:\/\/)((http[s]?\:\/\/)?([a-z0-9\.]+\.[a-z]{2,5}(\/[\?=&a-z0-9_\.\/]+)?)))
```

Comment:
In general, I haven't found kramdown automatic solution. Their offer to wrap links in <>, e.g.
`<github.com>` and it would be parsed as a link. We can offer users to do it in the same way